ext/on-modify.timewarrior

```diff
@@ -53,16 +65,19 @@ elif 'start' not in new and 'start' in old:
     start_or_stop = 'stop'
 
 if start_or_stop:
-    # Extract attributes for use as tags.
-    tags = [new['description']]
+    combined = get_timew_name_from_json(new)
 
-    if 'project' in new:
-        tags.append(new['project'])
+    command = 'timew {0} {1} :yes'.format(start_or_stop, combined)
+    subprocess.call(shlex.split(command))
 
-    if 'tags' in new:
-        tags.extend(new['tags'])
+# Modifications to task other than start/stop
+elif 'start' in new and 'start' in old:
+    old_combined = get_timew_name_from_json(old)
+    new_combined = get_timew_name_from_json(new)
 
-    combined = ' '.join(['"%s"' % tag for tag in tags]).encode('utf-8').strip()
+    if old_combined != new_combined:
+        command = 'timew stop {0} :yes'.format(old_combined)
```

Comment:
Is stopping the old task and starting a new task the right way to react to a modification? When you modify a running task, wouldn't it be better to remove the old tags and add the new ones?

The resulting call to Timewarrior would then be
```
timew untag @1 <old_combined>
timew tag @1 <new_combined>
```